creativecloud/blocks/interactive-marquee/interactive-marquee.js

```diff
@@ -71,6 +71,11 @@ export default async function init(el) {
   const { decorateButtons, decorateBlockBg } = await import(`${miloLibs}/utils/decorate.js`);
   const { createTag, loadStyle } = await import(`${miloLibs}/utils/utils.js`);
   switch (true) {
+    case el.classList.contains('changebg'): {
+      const { default: changeBg } = await import('../../features/changeBg/changeBg.js');
+      changeBg(el);
+      break;
+    }
     case el.classList.contains('genfill'): {
       loadStyle('/creativecloud/blocks/interactive-marquee/milo-marquee.css');
```

Comment:
@suhjainadobe The PR intended for dexter consumption will need the changes to dynamically load JS and CSS from config.origin (if set) or else from relative domain. Could you make the required changes for the same?